app/classifier/drawing-tools/point-grid.jsx

```diff
@@ -0,0 +1,101 @@
+const DeleteButton = require('./delete-button');
+
+const DELETE_BUTTON_ANGLE = 45;
+const SELECTED_RADIUS = {
+  large: 20,
+  small: 10
+};
+
+class PointGridTool extends React.Component {
+
+  static defaultValues({x, y}) {
+    return {x, y};
+  }
+
+  static initStart({x, y}) {
+    return {x: x, y: y, _inProgress: true};
+  }
+
+  static initRelease() {
+    return {_inProgress: false}
+  }
+
+  static initValid(mark, props) {
+    const naturalWidth = props.naturalWidth
+    const naturalHeight = props.naturalHeight
+    const offset_x = Number.parseInt(props.task.tools[mark.tool].x_offset)
+    const offset_y = Number.parseInt(props.task.tools[mark.tool].y_offset)
+
+    const notBeyondWidth = mark.x < naturalWidth;
+    const notBeyondHeight = mark.y < naturalHeight;
+    const beyondOffset = mark.x > offset_x && mark.y > offset_y
+
+    return notBeyondWidth && notBeyondHeight && beyondOffset;
+  }
+
+  static options = ['grid']
+
+  static defaultProps = {
+    rows: 10,
+    cols: 10,
+    offset_x: 50,
+    offset_y: 50,
+    opacity: 50,
+  }
+
+  getDeleteButtonPosition() {
+    const theta = (DELETE_BUTTON_ANGLE) * (Math.PI / 180);
+    return {
+      x: (SELECTED_RADIUS.large / this.props.scale.horizontal) * Math.cos(theta),
+      y: -1 * (SELECTED_RADIUS.large / this.props.scale.vertical) * Math.sin(theta)
+    };
+  }
+
+  render() {
+    const { offset_x } = this.props;
+    const { offset_y } = this.props;
+
+    const width = ((this.props.containerRect.width / this.props.scale.horizontal)-offset_x) / this.props.rows; 
+    const height = ((this.props.containerRect.height / this.props.scale.vertical) - offset_y) / this.props.cols;
+
+    let x = (this.props.mark.x-offset_x);
+    let y = (this.props.mark.y-offset_y);
+
+    x = (Math.floor(x/width));
+    y = Math.floor(y/height);
+
+    x = (x*width)+offset_x;
+    y = (y*height)+offset_y;
+    
+    return (
+      <DrawingToolRoot tool={this} transform={`translate(${x}, ${y})`}>
+        <rect x="0" y="0" width={width}
+          height={height}
+          fill={this.props.color}
+          fillOpacity={this.props.opacity / 100}
+          strokeOpacity="0"/>
+    {/*return React.createElement(DrawingToolRoot, tool_args,
+      React.createElement("rect", {"x": "0", "y": "0", "width": `${width}`, "height": `${height}`,  
+        "fill": `${this.props.color}`, "fillOpacity": `${this.props.opacity / 100}`, "strokeOpacity": "0"}),*/}
+
+        {!!this.props.selected &&
+          <DeleteButton tool={this} {...this.getDeleteButtonPosition()}  getScreenCurrentTransformationMatrix={this.props.getScreenCurrentTransformationMatrix} />
+        }
+      </DrawingToolRoot>
+    );
+  }
+
+  destroyTool = (e) => {
+    if (this.props.mark._inProgress == false && this.props.selected == true) {
+      return this.setState({destroying: true}, () => {
+        return setTimeout(this.props.onDestroy, 300);
+      });
+    }
+  }
+};
+
+module.exports = PointGridTool
```

Comment:
Change this to: `export default PointGridTool`

Reply:
I'm having trouble getting this to work. If I do this and remove the reference from the index.coffee file then this tool doesn't show up in the project builder
![screenshot](https://user-images.githubusercontent.com/10344353/37681811-b5d4e56a-2c55-11e8-86a3-a203ca9c969c.png)